### ISSUE TITLE
Path specified is not a VVV root directory

### ISSUE THREAD

**Issue by joecue** [bradp/vv#179]
On both my windows 7 desktop and OS X 10.10.5 desktop, I'm unable to use VV.  I attempt to run VV from bash and repeatedly have to set my path.

On my windows machine it will ask if this is where VV is installed, and ask to confirm.  I looked at the .vv.config file on my Win7 desktop and the file contained about 15 
"
{
"path":
}

"
On my Mac it just jumps right to the path statement.

If I try "vv list", windows will ask if this is where vv is, then the path to vvv, on the mac it just asks for the path to VVV, then both will actually list my VVV sites.

Haven't tried a create yet.

Curious thing is that my Win7 laptop that I've had VVV and vv setup on for a while, I needed to update vv, but it runs just fine.  Doesn't ask where vv is installed or VVV it just runs the command I ask.  

I can't figure out what the difference is between the three systems.  All three are reporting the same version, 1.9.3

Any ideas?

**Comment by bradp** [comment]
I'm not quite sure what the underlying issue is for these problems, but I have a few ideas.

The solution, however, is editing ~/.vv-config to contain the path to your VVV folder. On windows, you'll want to make sure its formatted like this: https://github.com/bradp/vv/issues/168#issuecomment-147413890

**Comment by joecue** [comment]
That didn't work.  When I tried to run vv on my Win7 Desktop, it asked again if this is where vv is installed and asked for the path.  When I tried to add the path formatted as you suggested, it threw an error.

On my laptop my path is "/c/vagrant/wp-sandbox" and that works just fine.  When I try to format my path the same on my desktop, it just keeps asking the same two questions.

This is very strange..

**Comment by bradp** [comment]
Make sure the permissions on ~/.vv-config are set to be readable / writable.

**Comment by joecue** [comment]
My user account as full control, as well as the administrators group and the system user.

**Comment by joecue** [comment]
after I tried to run vv the last and it asked the questions again, it added another { "path": ""} to the .vv-config file.

almost seems like the script isn't saving or writing to the config file correctly.

**Comment by joecue** [comment]
On my Mac the .vv-config in my users home directory is also empty just shows { "path": ""}.  The file is owned by my user and has -rw-r--r-- permissions, which I believe is correct for being readable and writeable.  Unlike the Win7 box though, an extra path didn't get added to the Mac after trying again.  I also edited the file in a text editor saved, and tried vv again, still was prompted for the location, but the file again didn't change from when I saved it in a text editor.

At least whatever is going on it's consistent between platforms.

**Comment by joecue** [comment]
The good news is that create does seem to work, it's provisioning a new site now... just annoying that I have to answer if this is where vv is installed and type the directory path each time.

**Comment by bradp** [comment]
This should be fixed in https://github.com/bradp/vv/pull/200

**Comment by joecue** [comment]
I ran "vv --update" it said it updated, but then vv wouldn't run.

I deleted the vv folder and re-cloned from github, and was able to get vv to run, but it's still asking where VVV is installed, it's at least finding the folder if I'm in the folder VVV is installed in, but it asks each time.

It's improved from where it was but still doesn't seem to persist.

This is on my Windows 7 machine, running vv from a bash window.
